### PR TITLE
Added a Project 

### PR DESCRIPTION
**Pull request by omk-coder**
Issue no: #30

### CODE REVIEW

**Comment by omk-coder** [comment]
hey I have worked on this issue  please take a look

File: projects/animesite/index.html [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+                        best-selling fiction authors..</p>
+                    <div class="rating">
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                    </div>
+                    <img
+                        src="http://t1.gstatic.com/licensed-image?q=tbn:ANd9GcQ-jN__eNpfqVbU1k-kifLJ45o1ht9mSneMDBmtX9G7ynCfb4uV-Cs6R4Upnnd4i3Em">
```
Comment (fykaa) [inline projects/animesite/index.html:282]: This link is causing mixed content conflict (https/http) please change it with some other source

Comment (fykaa) [reply projects/animesite/index.html:282]: @omk-coder

**Comment by fykaa** [comment]
LGTM, This PR will be accepted by hacktoberfest

**Comment by omk-coder** [comment]
Thanks @fykaa

**Pull request merged**